docker/README.md

```diff
@@ -0,0 +1,32 @@
+# Docker Integration
```

Comment:
We should probably state this check is deprecated and point people to `docker_daemon`.